ui/components/app/multiple-notifications/multiple-notifications.component.js

```diff
@@ -40,7 +40,7 @@ export default class MultipleNotifications extends PureComponent {
         >
           {childrenToRender.length > 1 ? (
             <i
-              className={classnames('fa fa-sm fa-sort-amount-asc', {
+              className={classnames('fa fa-sm fa-sort-amount', {
                 flipped: !showAll,
               })}
             />
```

Comment:
TODO! Find out what this component is. The icon class there doesn't show as it's not in the fontawesome library csss..

Reply:
You can see this component by doing this, and then looking at the bottom right of the home screen:

https://gist.github.com/darkwing/71b55629b9078afa7ce2e47ccd4e066f

Reply:
Ah ok yeah nothing shows up so I think this is fine

<img width="1440" alt="Screen Shot 2022-03-23 at 3 21 17 PM" src="https://user-images.githubusercontent.com/8112138/159806100-d5cfe148-81ba-4625-903a-35de90200ef8.png">